sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryProperties.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+import java.util.Map;
+
+/**
+ * Represents properties about a {@link Geometry}.
+ */
+public final class GeometryProperties {
+    private final GeometryBoundingBox boundingBox;
+    private final Map<String, Object> additionalProperties;
+
+    /**
+     * Constructs an empty properties.
+     */
+    public GeometryProperties() {
+        this(null, null);
+    }
+
+    /**
+     * Constructs properties about a {@link Geometry}.
+     *
+     * @param boundingBox The bounding box for a geometry.
+     * @param additionalProperties Addtional properties about a geometry.
+     */
+    public GeometryProperties(GeometryBoundingBox boundingBox, Map<String, Object> additionalProperties) {
+        this.boundingBox = boundingBox;
+        this.additionalProperties = additionalProperties;
```

Comment:
Should you make a copy of this map so that it can not be mutated after being set?

Reply:
Making a deep copy now.